pulser-core/pulser/register/register.py

```diff
@@ -127,6 +127,55 @@ def rectangle(
 
         return cls.from_coordinates(coords, center=True, prefix=prefix)
 
+    @classmethod
+    def rectangle_rect_lattice(
```

Comment:
I think you can use this method inside `rectangle()` now

Reply:
I'm not sure I understand what you mean.

Reply:
The contents of the method just above (`Register.rectangle()`)  can be rewritten to use this one since 
`Register.rectangle(rows, columns, spacing) == Register.rectangular_lattice(rows, columns, spacing, spacing)`